proj/src/main/webapp/static/signup.html

```diff
@@ -81,7 +94,7 @@ <h5 class="text-center"><a href="./welcome.html">already have an account? sign i
   <footer class="hide-on-mobile">
     <div>icons made by <a href="https://www.flaticon.com/authors/freepik" title="Freepik">freepik</a> from <a href="https://www.flaticon.com/" title="Flaticon">www.flaticon.com</a></div>
   </footer>
-  <script src="../chat.js"></script>
   <script src="../script.js"></script>
+  <script src="../chat.js"></script>
```

Comment:
was this necessary?

Reply:
yup chat.js have all of the datastore files and functions